Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:latest
+MAINTAINER nelluri@deis.com
+ENV CLOUD_SDK_REPO cloud-sdk-precise
+RUN echo "deb http://packages.cloud.google.com/apt cloud-sdk-precise main" | tee /etc/apt/sources.list.d/google-cloud-sdk.list
+RUN curl -L https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+RUN apt-get update && apt-get -y install make python lsb-release unzip wget vim git curl jq google-cloud-sdk
+RUN wget https://storage.googleapis.com/kubernetes-release/release/v1.0.3/bin/linux/amd64/kubectl && chmod +x kubectl && mv kubectl /usr/local/bin
```

Comment:
This should use a newer `kubectl` I think. Most of us test with v1.1.8: https://storage.googleapis.com/kubernetes-release/release/v1.1.8/bin/linux/amd64/kubectl